.circleci/config.yml

```diff
@@ -2,14 +2,14 @@ version: 2.1
 jobs:
   lint:
     docker:
-      - image: salsify/ruby_ci:2.5.8
+      - image: salsify/ruby_ci:2.6.6
```

Comment:
It shouldn't matter but since I had issues using the 2.6.6 image, can we change it to the 2.7.2 image instead?